src/components/Home/Ritika/index.tsx

```diff
@@ -1,14 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Link, Outlet } from 'react-router-dom';
+import { AppProvider } from './Week4/Context/context';
+import { MainContainerWrapper } from './styles';
+
+const Ritika: React.FC = () => {
+  const [selectedWeek, setSelectedWeek] = useState('');
 
-const Ritika = () => {
   return (
-    <>
-      <Link to='week2'>Week-2</Link>
-      <br></br>
-      <Link to='week3'>Week-3</Link>
-      <Outlet />
-    </>
+    <MainContainerWrapper>
+      <AppProvider>
+        <div className='weekButtonsContainer'>
+          <Link to='week2'>
```

Comment:
i feel like this code can be reduced if you use a map.

Reply:
Yes , thanks anushree i did it , now it will be more easy to just add week name and id and other things will automatically updated.